src/mono/mono/component/debugger-agent.c

```diff
@@ -2557,12 +2556,6 @@ suspend_vm (void)
 	}
 
 	mono_coop_mutex_unlock (&suspend_mutex);
-
-	if (suspend_count == 1)
-		/*
-		 * Suspend creation of new threadpool threads, since they cannot run
-		 */
-		tp_suspend = TRUE;
```

Comment:
this is no longer used because this code from mono/mono was undefined in ENABLE_NETCORE and removed in dotnet/runtime: https://github.com/mono/mono/blob/8b3b6666a858e02137b192c923727bcfd00ef620/mono/mini/debugger-agent.c#L2873-L2876

@lambdageek @vargaz do you know if that is intentional, i.e. do we no longer need to suspend creating threadpool threads? or is this just something that was missed while porting to dotnet/runtime now that the threadpool works differently?

Reply:
Maybe @vargaz remembers.  I _think_ we don't need it anymore - the old threadpool had a native Monitor thread that was responsible for monitoring workers.  So the debugger stopped all the managed worker threads, but it didn't have any idea about the monitor thread.  In netcore, we don't have a native monitor thread - everything is done by managed threads.  So when the debugger suspends the runtime, it will suspend the thread pool worker creation, too.

Reply:
Ok sounds good then :)